deployment/ansible/roles/model-my-watershed.geoprocessing/templates/upstart-akka-http.conf.j2

```diff
@@ -0,0 +1,13 @@
+description "Akka HTTP Server"
+
+start on runlevel [3]
+stop on shutdown
+
+kill timeout 20
```

Comment:
Hm. I may have put the rules in a bad state when testing this out.

Let me drop the 7f391d0, add `setuid mmw` to the upstart job, and see if it will works.